config/gulp/javascript.js

```diff
@@ -0,0 +1,55 @@
+var gulp = require( 'gulp' );
+var gutil = require( 'gulp-util' );
+var dutil = require( './doc-util' );
+var browserify = require( 'browserify' );
+var buffer = require( 'vinyl-buffer' );
+var source = require( 'vinyl-source-stream' );
+var uglify = require( 'gulp-uglify' );
+var sourcemaps = require( 'gulp-sourcemaps' );
+var rename = require( 'gulp-rename' );
+var assert = require( 'gulp-if' );
+var linter = require( 'gulp-eslint' );
+var task = /([\w\d-_]+)\.js$/.exec( __filename )[ 1 ];
+
+gulp.task( 'eslint', function ( done ) {
+
+  if ( ! cFlags.test ) {
```

Comment:
What JS styleguide are we using btw.

I have to be honest, I hate having this amount of space inbetween `()`